src/components/Photos/index.js

```diff
@@ -11,12 +11,149 @@ import { wallpaperPhotos } from './imgCategories/Wallpapers';
 
 import './Photos.css';
 
+const categories = {
+    "All": "",
+    "StuySquad": stuysquadPhotos,
+    "Sing": singPhotos,
+    "SOS": sosPhotos,
+    "Club": clubPhotos,
+    "Multigen": multigenPhotos,
+    "Wallpapers": wallpaperPhotos,
+    "Logos": logoPhotos,
+    "Merch": merchPhotos,
+    "Others": otherPhotos,
+};
+
+const sorts = [ "Date Asc.", "Date Desc." ];
+
+const get_photos = (category,sort) => {
+    if (category === "All") {
+	var c = Object.keys(categories)
```

Comment:
Please do not use `var` in React code. Change it to `const` if you don't plan on changing it or `let` if you want to change it later on in the code.
```javascript
const a = 3;
a = 4; // Error
let b = 3;
b = 4; // All is fine
```